app/commands/filter.py

```diff
@@ -0,0 +1,45 @@
+import sqlalchemy as sa
+from app import models as m
+from app.database import db
+
+# Run script example
+# flask filter-addresses 10 "a1e9f9b8-41b9-11df-907f-00215aee3ebe" "91fc81db-266d-11e7-80fd-1c98ec135263"
+
+
+def filter_addresses(region_id: int, district_id: str, city_id: str):
+    """Filter addresses"""
+
+    with db.begin() as session:
+        region = session.execute(sa.select(m.Region).filter_by(id=region_id)).scalar_one_or_none()
+        print("Region: ", region)
+        print("========================================================================================")
+
+        if not region:
+            print("Region not found")
+            session.close()
+            return
+
+        rayons = session.query(m.Rayon).filter(m.Rayon.location_id == region.id).all()
+        print("Rayons: ", rayons)
```

Comment:
redundant prints

Reply:
All these prints are intended for testing db after scraping api. Yes, you can remove them and only monitor the bad outcome where the entity is not found, but my idea is to check whether my filtering gives the same data as the filtering on the API itself.